### ISSUE TITLE
File paths have C:\ 2 times

### ISSUE THREAD

**Issue by Coconaut11** [KeitIG/museeks#354]
When i try to play a song, it gives me that error (with lastest update):
Uncaught Error: ENOENT: no such file or directory, open 'C:\C:\Users\Cocon\Music\Uptown Funk.mp3'
events.js:160

**Comment by KeitIG** [comment]
It gave it to me too, the music is still playing though, right ?

**Comment by Coconaut11** [comment]
No, it doesn't play, visually, it seems like it would be playing, but it doesn't.

**Comment by KeitIG** [comment]
ok, really weird, it does work on my side. Checking now.

**Comment by KeitIG** [comment]
@Coconaut11 Could you try again with some other tracks ? By my side, I get the error, but the music is still playing, so I suspect a problem with Chrome.

If it still doesn't play, can you send me the full path of the located file you want to play?

**Comment by Coconaut11** [comment]
It doesn't matter which track i try, or where it is, I tried moving the tracks to other folders, but still doesn't work.

![captura](https://user-images.githubusercontent.com/15111802/28755009-f8b58616-7551-11e7-9691-17d3de6e0b1e.PNG)

**Comment by KeitIG** [comment]
Thanks, will investigate. Seems pretty urgent.

**Comment by KeitIG** [comment]
Can you tell me which version of Windows you are using ?

**Comment by Coconaut11** [comment]
Windows 10

**Comment by YurySolovyov** [comment]
Might be related https://github.com/nodejs/node/pull/15490

**Comment by KeitIG** [comment]
@Coconaut11 can you test the latest version of Museeks and tell me if you can reproduce it?
